darwin/DarwinMachine.c

```diff
@@ -125,3 +135,34 @@ bool Machine_isCPUonline(const Machine* host, unsigned int id) {
 
    return true;
 }
+
+void Machine_scanGPUUsage(Machine* super) {
+   DarwinMachine* dhost = (DarwinMachine *)super;
+   CFMutableDictionaryRef properties = NULL;
+   CFDictionaryRef perfStats;
+   CFNumberRef deviceUtil;
+   int device = 0, ret;
+
+   if (!dhost->GPUService) {
+      return;
+   }
+
+   ret = IORegistryEntryCreateCFProperties(dhost->GPUService, &properties, kCFAllocatorDefault, kNilOptions);
+   if (ret != KERN_SUCCESS) {
+      return;
+   }
+
+   perfStats = CFDictionaryGetValue(properties, CFSTR("PerformanceStatistics"));
```

Comment:
Ditto. Declare variables just before their first use.
```suggestion
   CFDictionaryRef perfStats = CFDictionaryGetValue(properties, CFSTR("PerformanceStatistics"));
```